### ISSUE TITLE
Image Preprocessing for stated top5 accuracy

### ISSUE THREAD

**Issue by fervorarc** [DeepScale/SqueezeNet#23]
What image preprocessing was performed on the imagenet images to achieve the stated feedforward top5 accuracy?
E.g. resize uniformly to 256 at the smallest dimension, then center crop 
I've had a tough time figuring this out, and any help would be much appreciated. 
Many thanks!

**Comment by Cospel** [comment]
Hi, that will be great.

I cannot find this in the paper. Can authors please respond. I am trying to implement and train it from scratch in Tensorflow, but I am not sure what strategy did you used.

**Comment by itdxer** [comment]
I found this information specified in caffe training file here:
https://github.com/DeepScale/SqueezeNet/blob/master/SqueezeNet_v1.0/train_val.prototxt

```
  transform_param {
    crop_size: 227
    mean_value: 104
    mean_value: 117
    mean_value: 123
}
```
